frame/fast-unstake/src/lib.rs

```diff
@@ -109,12 +106,7 @@ pub mod pallet {
 	pub struct Pallet<T>(_);
 
 	#[pallet::config]
-	pub trait Config:
-		frame_system::Config
-		+ pallet_staking::Config<
-			CurrencyBalance = <Self as pallet_nomination_pools::Config>::CurrencyBalance,
-		> + pallet_nomination_pools::Config
-	{
+	pub trait Config: frame_system::Config + pallet_staking::Config {
```

Comment:
Later on, you can also tackle another part of #12337 and implement this via `trait StakingInterface` (if it supports all the needed APIs).

Reply:
will do